_posts/2016-10-31-angular2-discussion-portal.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title:  "Angular 2 application with refreshing data in real time"
+date:   2016-10-31 08:00:00 +0200
```

Comment:
you can update date to the future, maybe we can publish this on monday or tuesday